### PR TITLE
UI: explicitly specify namespace

### PR DESCRIPTION
**Pull request by bershanskiy**
### Description

Remove all occurences of `using namespace std;` and explicitly specify `std::` namespace.
Replace all occurences of `using namespace json11;` with `using json11::Json;` which acheives the same thing.

### Motivation and Context

Code cleanup:
 - consistency with other OBS code;
 - (subjectively) easier to read;
 - (incidentally) less LOC;

### How Has This Been Tested?

It compiles on CI and produces the same artifacts as before this commit (I'll post detailed comparison of CI artifacts today).

### Types of changes

 - Code cleanup (non-breaking change which makes code smaller or more readable)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: UI/window-basic-main.cpp [hunk 1]
```diff
@@ -152,24 +151,25 @@ static void AddExtraModulePaths()
 
 	char base_module_dir[512];
 #if defined(_WIN32) || defined(__APPLE__)
-	int ret = GetProgramDataPath(base_module_dir, sizeof(base_module_dir),
-				     "obs-studio/plugins/%module%");
+	int ret =
+		GetProgramDataPath(base_module_dir, sizeof(base_module_dir),
+				   "light-studio/obs-studio/plugins/%module%");
```
Comment (WizardCM) [inline UI/window-basic-main.cpp:156]: This seems unrelated to the PR. There are 13+ instances of this change.

**Review by WizardCM — CHANGES_REQUESTED** [review]
I feel this change is overall unnecessary, but I'll let others provide their thoughts.

**Comment by notr1ch** [comment]
This feels like it only makes it more annoying to work on UI code that makes heavy use of std. If there's no namespace conflicts I don't really see this being necessary.

File: UI/window-basic-main.cpp (hunk 1 above)
Comment (bershanskiy) [reply UI/window-basic-main.cpp]: Thanks for catching this, I didn't mean for this change to be included. I should have marked PR as a draft.
